DataStructures/SortedLinkedList/SortedLinkedListNode.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace DataStructures.SortedLinkedList
+{
+    /// <summary>
+    /// TODO.
```

Comment:
Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6332606)